src/components/SimpleTable.js

```diff
@@ -90,8 +90,10 @@ const DataRows = ({rowData, baseStyle, columnHeaders}) => {
         }
         return output;
       }));
+
+    return rowWrapper({renderedRow, rowData: row});
```

Comment:
I added the table here by adding a function which can inject HTML around a row. I considered a lot of different options, but this seemed the cleanest / least invasive way of achieving adding code inside of the outer DAR collection table itself.